omego/db.py

```diff
@@ -14,6 +14,43 @@
 
 log = logging.getLogger("omego.db")
 
+# Regular expression identifying a SQL schema
+SQL_SCHEMA_REGEXP = re.compile('.*OMERO(\d+)(\.|A)?(\d*)([A-Z]*)__(\d+)$')
```

Comment:
I can either include this change as part of this PR of a follow-up one (since this PR tries to solve an immediate deployment issue). One question here is about the schemas ordering logic i.e. what are the rules to compute the order listed above (we have one more discriminator tahn the previous `M.m__p`)?